caffe2/mobile/contrib/arm-compute/core/operator.cc

```diff
@@ -0,0 +1,9 @@
+#include "operator.h"
+
+namespace caffe2 {
+
+CAFFE_DEFINE_REGISTRY(GLOperatorRegistry, OperatorBase, const OperatorDef &,
+                      Workspace *);
+CAFFE_REGISTER_DEVICE_TYPE(DeviceType::OPENGL, GLOperatorRegistry);
```

Comment:
This probably collides with our OpenGL ES 3.0 backend

Reply:
Is OpenGL ES 3.0 backend removed already?

Reply:
No, its still in the source tree, and potentially even being built.

Reply:
Should we change it to something else? I think DeviceType::OPENGL is a pre-defined device type.

Reply:
cc @hlu1 to check if this can conflict with OpenGL ES 3.0

Reply:
```
#ifdef CAFFE2_OPENGL_BACKEND
#error Can only build one OpenGL backend at a time.
#else
#define CAFFE2_OPENGL_BACKEND
#endif
```

Reply:
OpenGL ES 3.0 backend does not use DeviceType. It prefixes operator names with OpenGL instead